packages/models-library/src/models_library/projects.py

```diff
@@ -113,6 +113,13 @@ def convert_sql_alchemy_enum(cls, v):
     )
 
 
+def _patch_json_schema_extra(schema: dict) -> None:
+    # Patch to allow jsonschema nullable
+    # SEE https://github.com/samuelcolvin/pydantic/issues/990#issuecomment-645961530
+    state_pydantic_schema = deepcopy(schema["properties"]["state"])
+    schema["properties"]["state"] = {"anyOf": [{"type": "null"}, state_pydantic_schema]}
```

Comment:
I thought this was already in the new version!?

Reply:
If we derive the `json_schema` for a `Project`, this is what happens "batteries included" in **Pydantic v2** for the optional `state` field:

```json
"state": {
      "anyOf": [
        {
          "$ref": "#/$defs/ProjectState"
        },
        {
          "type": "null"
        }
      ],
      "default": null
    },
```

So, **yes**.